rpc/wrtc_client.go

```diff
@@ -233,7 +234,6 @@ func dialWebRTC(
 				return
 			}
 			if icecandidate != nil {
-				logger.Debugw("gathered local ICE candidate", "candidate", icecandidate.String())
```

Comment:
I don't think this log nor the corresponding remote log below are important. We should just look the `Connection state changed` logs to see timestamps of when local or remote candidates were added to the peer connection.

Reply:
It sounds like the underlying timestamp of when candidates are gathered is already recorded/surfaced by pion. Agreed we no longer need this.